src/app/public/modules/autocomplete/autocomplete-default-search-function.ts

```diff
@@ -39,15 +39,15 @@ export function skyAutocompleteDefaultSearchFunction(
     data: any[]
   ): SkyAutocompleteSearchFunctionResponse {
 
-    const searchTextLower = searchText.toLowerCase();
+    const searchTextUpper = searchText?.toUpperCase();
```

Comment:
This was not a part of this bug; however, I noticed this while in here and updated based on Paul's advice a while back in standup. 

(The null check was found during this change)